extensions/amp-analytics/0.1/vendors/blueconic.json

```diff
@@ -0,0 +1,10 @@
+{
+  "requests": {
+    "event": "https://${tenantId}/s/${pixelId}.gif?${trackingIdentifier}${defaultTrackingValues}${extraTrackingValues}"
```

Comment:
`tenantId` and `pixelId` is not defined in `vars` in this json, so they will resolve to null.

Reply:
This is intended, we will use an config.json to fill in the blanks. Because each customer has his own environment.

Reply:
Can you explain the process a bit more?

Reply:
All customers have their own tenant. For each tenant the customer can configure their own events they want to track. This results in a config.json that will be retrieved from the customers tenant by providing this link in the tag:
```
<amp-analytics 
  type="blueconic"
  id="blueconic" 
  config="https://tenant01-slot05-bc05.blueconic.net/rest/custom/frontend/listener_amp/config?itemId=6001612d-a716-4379-8882-b648b336b49a"
>
</amp-analytics>
```
An example of the config file:
```
{
    "triggers": {
      "trackPageview": {
        "on": "visible",
        "request": "pageview",
        "vars": {
          "pixelId": "3c",
          "extraTrackingValues": "&title=${title}" // example of usage
        }
      },
      "click on #header trigger": {
        "on": "click",
        "selector": "#position_1",
        "request": "event",
        "vars": {
          "pixelId": "3d"
        }
      }
    },
    vars: {
      "tenantId": "tenant01-slot05-bc05.blueconic.net",
      "property": "amp_client_id" // this one is configurable
    }
  }
```

I hope this explains the process.

Reply:
Perfect, sounds great :) Thanks for the explanation